### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by gforcada**

### CODE REVIEW

**Comment by gforcada** [comment]
@tisto seems that something is off, if I follow the steps that I mention I end up with a non-working solr core, as for whatever reason stopwords.txt is not found :disappointed: although is correctly copied to parts/ along with all the config folder...

File: README.rst [hunk 1]
```diff
@@ -31,10 +31,9 @@ Apache Solr is based on Lucene and is *the* enterprise open source search engine
 It powers the search of sites like Twitter,
 the Apple and iTunes Stores, Wikipedia, Netflix and many more.
 
-Solr does not only scale to any level of content,
-but provides rich search functionality,
-like faceting, geospatial search, suggestions, spelling corrections, indexing of binary formats and a whole variety of powerful tools to configure custom search solutions.
-It has integrated clustering and load-balancing to provide a high level of robustness.
+For the wide range of functionality provided by Solr, see `Solr features`_.
```
Comment (tisto) [inline README.rst]: @gforcada linking to this page is a good idea. Though, I would somehow like to still list the main Solr features for convenience.

File: README.rst [hunk 2]
```diff
@@ -68,52 +67,55 @@ Search Enhancements
 Detailed Documentation
 ======================
 
-A full Documentation of the Solr integration of Plone could be found on `collectivesolr.readthedocs.org`_.
+The full documentation of Plone integration with Solr can be found at `read the docs`_.
 
-.. _`collectivesolr.readthedocs.org`: http://collectivesolr.readthedocs.org/en/latest/
+.. _`read the docs`: https://collectivesolr.readthedocs.io/en/latest/
```
Comment (tisto) [inline README.rst]: @gforcada I kind of prefer to keep the full URL. Some people might not know what readthedocs is.

File: README.rst [hunk 3]
```diff
@@ -68,52 +67,55 @@ Search Enhancements
...
-  collective.recipe.solrinstance = 5.3.2
-
 After saving this to let's say ``buildout.cfg``,
 the buildout can be run and the `Solr`_ server and `Plone`_ instance started::
 
-  $ python bootstrap-buildout.py
-  $ bin/buildout
+  $ virtualenv .
+  $ . bin/activate
+  $ pip install setuptools==26.1.1 zc.buildout==2.5.3
```
Comment (tisto) [inline README.rst]: @gforcada this will be outdated soon. :/

File: README.rst [hunk 4]
```diff
@@ -155,13 +157,17 @@ and require a good deal of memory from ZODB caches.
...
+Recommended is the latest in the Plone 5.0 series, currently 5.0.8.
 
 Solr Compatibility
 ==================
 
-The code works with Solr 3 or 4.
-Solr 5 is not yet supported.  See https://github.com/collective/collective.solr/issues/66
-Recommended is the latest in the Solr 4 series, currently 4.10.4.
+The code works with Solr 7.
+Recommended is the latest in the Solr 7 series, currently 7.4 and 7.5.
```
Comment (tisto) [inline README.rst]: @gforcada and now Solr 7.6. Maybe we should just mention 7.x for now.

**Pull request closed without merging**